### PR TITLE
Add support for multiline string values

### PR DESCRIPTION
**Pull request by maguro**
- backticks, `, and triple quotes, “””, now allow multiline strings
- splits STRING into STRING_NAME and STRING_VALUE

### CODE REVIEW

**Comment by maguro** [comment]
This is for "[FR: Text proto format could support multi-line strings](https://github.com/protocolbuffers/protobuf/issues/1297)", "[Add support for multi-line strings](https://github.com/protostuff/protostuff-compiler/issues/60)", and "[Add support for multiline strings](https://github.com/protostuff/protobuf-jetbrains-plugin/issues/139)".

File: build.gradle [hunk 1]
```diff
@@ -33,7 +33,7 @@ dependencies {
         exclude group: 'org.slf4j'
     }
     compile 'com.google.guava:guava:21.0'
-    compile ('io.protostuff:protostuff-parser:2.2.26') {
+    compile ('io.protostuff:protostuff-parser:0-SNAPSHOT') {
```
Comment (maguro) [inline build.gradle]: I'll wait until this gets released to fix this.

Comment (kshchepanovskyi) [reply build.gradle]: sure

**Comment by kshchepanovskyi** [comment]
This issue is closed as plugin is not supported anymore. 
Please switch to https://github.com/jvolkman/intellij-protobuf-editor.

**Pull request closed without merging**